pkgs/applications/editors/vim/plugins/vim-utils.nix

```diff
@@ -149,13 +149,12 @@ let
     # make sure all the dependencies of the plugin are also derivations
     drv // { dependencies = map (pluginToDrv knownPlugins) (drv.dependencies or []); };
 
-  # transitive closure of plugin dependencies (plugin needs to be a derivation)
-  transitiveClosure = plugin:
-    [ plugin ] ++ (
-      lib.unique (builtins.concatLists (map transitiveClosure plugin.dependencies or []))
-    );
+  reverseUniq = list: lib.reverseList (lib.unique (lib.reverseList list));
```

Comment:
Why do we need to reverse list?

Reply:
lib.unique keeps the first same element but I want to keep the last one.